app/run-5-postprocess.sh

```diff
@@ -12,5 +12,6 @@ echo -e "\e[1m\e[7m Postprocessing – START \e[27m\e[21m\e[0m"
 # unlock database
 
 psql -q -f "./POSTPROCESS.sql"
+psql -q -f "./analysis/bikelanes.sql"
```

Comment:
Ja, ich würde es in einen eigenen Run packen, das macht es übersichtlicher im logging in den logfiles und im chat.